handler/src/main/java/io/netty/handler/ssl/SslContextBuilder.java

```diff
@@ -464,4 +533,15 @@ public SslContext build() throws SSLException {
                 ciphers, cipherFilter, apn, protocols, sessionCacheSize, sessionTimeout, enableOcsp, keyStoreType);
         }
     }
+
+    private static <T> T[] toArray(Iterable<? extends T> iterable, T[] prototype) {
+        if (iterable == null) {
+            return null;
+        }
+        final List<T> list = new ArrayList<T>();
```

Comment:
Probably not a big deal given it's unlikely to be called frequently, but would be nice to check for `instanceof Collection` and then use `.toArray(...)` directly instead of copying to new list.